lib/util/sdp/issue8329.js

```diff
@@ -87,6 +88,43 @@ function mediaSectionWorkaround(mediaSection) {
   return mediaSection;
 }
 
+/**
+ * @param {string} mediaSection
+ * @param {Map<PT, Codec>} ptToCodecName
+ * @returns @param {string} newMediaSection
+ */
+function deleteDuplicateRtxPts(mediaSection, ptToCodecName) {
```

Comment:
@syerrapragada Can we generalize this to remove duplicate rtp payload type mappings irrespective of whether they map to `rtx` or not?